src/pyff/test/data/metadata/test-sp.xml

```diff
@@ -0,0 +1,906 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<md:EntitiesDescriptor xmlns:md="urn:oasis:names:tc:SAML:2.0:metadata" xmlns:ds="http://www.w3.org/2000/09/xmldsig#" xmlns:mdui="urn:oasis:names:tc:SAML:metadata:ui" xmlns:mdattr="urn:oasis:names:tc:SAML:metadata:attribute" xmlns:mdrpi="urn:oasis:names:tc:SAML:metadata:rpi" xmlns:shibmd="urn:mace:shibboleth:metadata:1.0" xmlns:xrd="http://docs.oasis-open.org/ns/xri/xrd-1.0" xmlns:pyff="http://pyff.io/NS" xmlns:saml="urn:oasis:names:tc:SAML:2.0:assertion" xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:ser="http://eidas.europa.eu/metadata/servicelist" xmlns:eidas="http://eidas.europa.eu/saml-extensions" xmlns:alg="urn:oasis:names:tc:SAML:metadata:algsupport" xmlns:samla="urn:oasis:names:tc:SAML:2.0:assertion" xmlns:init="urn:oasis:names:tc:SAML:profiles:SSO:request-init" xmlns:idpdisc="urn:oasis:names:tc:SAML:profiles:SSO:idp-discovery-protocol" xmlns:wayf="http://sdss.ac.uk/2006/06/WAYF" xmlns:req-attr="urn:oasis:names:tc:SAML:protcol:ext:req-attr" Name="test">
+    <md:Extensions>
+      <ti:TrustInfo xmlns:ti="https://seamlessaccess.org/NS/trustinfo">
```

Comment:
The spec says
> The extension is intended to be added to the Extensions element of the SPSSODescriptor of an EntityDescriptor used to describe a SAML service provider.
but this `ti:TrustInfo` element is in the extensions of the `md:EntitiesDescriptor`. Should it be here?

Reply:
This is so that federations can provide a number of globally available ti:TrustProfile elements that specific SP operators can use via ti:TrustProfileRef's.

Reply:
Sorry, I missed the paragraph where you talk about "When the extension is present in the Extension element of the EntitiesDescriptor..."